src/js/elements/menu.js

```diff
@@ -5,22 +5,10 @@ const hideondeployClsName = 'hideondeploy';
 const parentContainer = $(".data-mapper-content__list");
 let categoryTemplate = null;
 let subCategoryTemplate = null;
-let indicatorTemplate = null;
```

Comment:
@adieyal is there a reason you removed indicatorTemplate declaration? this let to the reference error